tests/test_multisim_fixed_source/test_multisim_fixed_source.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import hashlib
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import MultiSimTestHarness
+from input_set import HomInfMedInput
+import openmoc
+import openmoc.process
+
+
+class MultiSimFixedSourceTestHarness(MultiSimTestHarness):
+    """A multi-simulation fixed source calculation for a cube of water with
+    7-group cross sections and a fixed box source."""
+
+    def __init__(self):
+        super(MultiSimFixedSourceTestHarness, self).__init__()
+        self.input_set = HomInfMedInput()
+        self.res_type = openmoc.TOTAL_SOURCE
+        self.solution_type = 'source'
+        self.source_cell = None
+        self.fluxes = []
+
+    def _create_geometry(self):
+        """Put a box source in the cube."""
+
+        self.input_set.create_materials()
+        self.input_set.create_geometry()
+
+        # Get the root Cell
+        cells = self.input_set.geometry.getAllCells()
+        for cell_id in cells:
+            cell = cells[cell_id]
+            if cell.getName() == 'root cell':
+                root_cell = cell
+
+        # Apply VACUUM BCs on all bounding surfaces
+        surfaces = root_cell.getSurfaces()
+        for surface_id in surfaces:
+            surface = surfaces[surface_id]._surface
+            surface.setBoundaryType(openmoc.VACUUM)
+
+        # Replace fissionable infinite medium material with C5G7 water
+        self.materials = \
+            openmoc.materialize.load_from_hdf5(filename='c5g7-mgxs.h5',
+                                               directory='../../sample-input/')
+
+        lattice = openmoc.castUniverseToLattice(root_cell.getFillUniverse())
+        num_x = lattice.getNumX()
+        num_y = lattice.getNumY()
+        width_x = lattice.getWidthX()
+        width_y = lattice.getWidthY()
+
+        # Create cells filled with water to put in Lattice
+        water_cell = openmoc.Cell(name='water')
+        water_cell.setFill(self.materials['Water'])
+        water_univ = openmoc.Universe(name='water')
+        water_univ.addCell(water_cell)
+
+        self.source_cell = openmoc.Cell(name='source')
+        self.source_cell.setFill(self.materials['Water'])
+        source_univ = openmoc.Universe(name='source')
+        source_univ.addCell(self.source_cell)
+
+        # Create 2D array of Universes in each lattice cell
+        universes = [[[water_univ]*num_x for _ in range(num_y)]]
```

Comment:
Actually no - this will produce a faulty result which is why I used this notation in the equivalent "point-source" sample input file. The reason is that multiplying a nested list actually creates a list of pointers. For example, the current notation will produce the following homogeneous array of cells with a tiny green fixed source cell:

![cells-z-0](https://cloud.githubusercontent.com/assets/209492/13083360/a6ef2a36-d4a2-11e5-9083-e6629f2f0800.png)


Your simpler notation however would produce the following vertical line of fixed source cells:

![cells-z-0](https://cloud.githubusercontent.com/assets/209492/13083325/7af26966-d4a2-11e5-8a6d-ffb8f5294dac.png)